apps/web-giddh/src/app/vat-report/vatReport.component.html

```diff
@@ -1,49 +1,19 @@
 <section appTranslate [file]="'vat-report'" (localeData)="localeData = $event"
     (commonLocaleData)="commonLocaleData = $event">
-    <nav class="navbar">
```

Comment:
Do check the `nav` opening tag is removed but can't find the closing `</nav>` tag getting removed in diff

Reply:
Added